src/functionalTest/resources/scenarios/RIA-3290-sample-request-for-send-notification.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This test is only covering the happy case scenario where we get a 200 response.  Have we covered the 400 / 500 scenarios also?  This may be another task in another ticket to do later since validation is the priority now.

Reply:
This will be addressed in PR #77